json/src/lib.rs

```diff
@@ -1039,6 +1039,23 @@ pub struct FundRawTransactionResult {
     pub change_position: i32,
 }
 
+#[derive(Deserialize, Clone, PartialEq, Eq, Debug)]
+pub struct GetBalancesResultEntry {
+    #[serde(with = "bitcoin::util::amount::serde::as_btc")]
+    pub trusted: Amount,
+    #[serde(with = "bitcoin::util::amount::serde::as_btc")]
+    pub untrusted_pending: Amount,
+    #[serde(with = "bitcoin::util::amount::serde::as_btc")]
+    pub immature: Amount,
+}
+
+#[derive(Deserialize, Clone, PartialEq, Eq, Debug)]
+#[serde(rename_all = "camelCase")]
```

Comment:
What's the reason for renaming to camel case? [According to the docs](https://bitcoincore.org/en/doc/0.20.0/rpc/wallet/getbalances/) the RPC data structures use snail case. But I see that we do this for other structures too, so I'm a bit confused.

Anyway, if it's useful it should probably be added to `GetBalancesResultEntry` too.